Proposed_First_FLEDGE_OT_Details.md

```diff
@@ -67,6 +67,16 @@ The FOT#1 will include event-level reporting for both winning and losing bids. I
 
 If the bid being generated or scored loses the auction, the URL will be fetched. These worklets may also call a `forDebuggingOnly.reportAdAuctionWin()` API which operates similarly to `forDebuggingOnly.reportAdAuctionLoss()` API but only fetches the URL after a winning bid or score.
 
+If a seller rejected a bid from participating in an auction, the seller can optionally add a rejectReason field to its return object. The buyer can learn about the reject reason through calling `forDebuggingOnly.reportAdAuctionLoss()` API in  `generateBid()`, by using “${rejectReason}” in the report URL's query string. The reject reason returned by `scoreAd()` should be one of
+*   "not-available"
+*   "invalid-bid"
+*   "bid-below-auction-floor"
+*   "pending-or-disapproved-by-exchange"
+*   "blocked-by-publisher"
+*   "language-exclusions"
+*   "category-exclusions"
+The default value will be "not-available", in cases where the seller doesn't provide one reject reason, or when a bid was not rejected.
```

Comment:
Mentioned this in the Chrome CL, but to ask more publicly:

Should we provide this information to seller debug reports as well?  I guess they have the information when they're assembling the debug report URL, anyways, except in the case of component auction, when the top-level seller rejects a bid that goes through a component seller.  Should the component seller have the information in that case?

Also, should we mention that in component auctions, the value can come from whichever seller rejected the bid?  I believe that's the current behavior.

Reply:
Updated the PR to describe the case of component auction bidders.

About the seller debug reports part, maybe we can post  under [274](https://github.com/WICG/turtledove/issues/274), to get more comments on it?